proto/order.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+
+package order;
```

Comment:
if we need this? how about just in `common.proto`

Reply:
Ok, will move them.